### PR TITLE
dx-1139 Added info about guest checkout.

### PR DESCRIPTION
**Pull request by arebra**

### CODE REVIEW

File: checkout/checkin.md [hunk 1]
```diff
@@ -31,6 +31,9 @@ The payer will be identified with the `consumers` resource and will be
 persisted to streamline future Payment Menu processes. Payer identification
 is done through the `initiate-consumer-session` operation.
 
+If the end user is using the Payment Menu as a guest, you can go directly to
+step 3, which you will find on the next page.
```
Comment (asbjornu) [inline checkout/checkin.md:35]: Turn this into an informational alert so it's more visible, perhaps?

**Review by asbjornu — CHANGES_REQUESTED** [review]
Marvelous work!

File: checkout/checkin.md (hunk 1 above)
Comment (arebra) [reply checkout/checkin.md]: Done!

File: checkout/checkin.md [hunk 2]
```diff
@@ -21,16 +21,20 @@ sidebar:
...
 
 ## Step 1: Initiate session for consumer identification
 
 The payer will be identified with the `consumers` resource and will be
 persisted to streamline future Payment Menu processes. Payer identification
 is done through the `initiate-consumer-session` operation.
 
+{% include alert.html type="informative" icon="info" body="
+Note: If the end user is using the Payment Menu as a guest, you can go directly 
+to step 3, which you will find on the next page." %}
```
Comment (asbjornu) [inline checkout/checkin.md:36]:
```suggestion
{% include alert.html type="informative" icon="info" header="Guest Checkout"
body="Note: If the end user is using the Payment Menu as a guest, you can go
directly to step 3, which you will find on the next page." %}
```

**Review by asbjornu — CHANGES_REQUESTED** [review]

**Review by marinafrydenb — APPROVED** [review]

**Review by asbjornu — APPROVED** [review]

**Pull request merged**